.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,29 +1,34 @@
-#### Code Sample, a copy-pastable example if possible
+---
+name: Bug Report
+about: Create a bug report to help us improve pandas
+title: "BUG:"
+labels: ""
+
+---
+
+#### Code Sample, a copy-pastable example
 
 ```python
 # Your code here
 
 ```
+
+**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.
+
 #### Problem description
 
-[this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
+[this should explain **why** the current behaviour is a problem and why the expected output is a better solution]
 
 **Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before. Please check first before submitting so that we do not have to handle and close duplicates!
```

Comment:
As a suggestion. I think we should change these two ` **Note** ` sections into checkboxes like

```
- [ ] I have checked that issue has not already been reported
- [ ] I have confirmed this bug exists on the latest version of pandas
- [ ] I have confirmed this bug exists on the master branch of pandas
```

Inspired how Numba does their bug reports: https://github.com/numba/numba/issues/4735